### DISCUSSION TITLE
Location of files

### DISCUSSION THREAD

**Discussion by angr72** [matt8707/hass-config#153] [General]
Where should these files be copied to:

Manually copy over these files from [matt8707/hass-config](https://github.com/matt8707/hass-config)

ui-lovelace.yaml
button_card_templates folder
popup folder
themes.yaml
sidebar.yaml

**Comment by matt8707** [comment]
All files should be placed in your configuration folder, where configuration.yaml is. 

Do you have any notes how this can be explained better in [INSTALL.md](https://github.com/matt8707/hass-config/blob/master/INSTALL.md)?

<img src="https://user-images.githubusercontent.com/36163594/213025276-b8b7b3e3-0ac8-4c43-949f-dec6dcc95664.jpg" width="30%" >

**Reply by angr72** [reply]
Yepp I can type something as soon as i get it to work :-)

**Reply by angr72** [reply]
Manually copy over these files from matt8707/hass-config 
 
Place the files and folders in the \\homeassistant.local\config folder 
 
Files: 

- ui-lovelace.yaml 
- sidebar.yaml 

 
Folders, might already exists but overwrite: 

- button_card_templates 
- popup 
- Themes 
- Www 

 
 
Select tablet and dark in Theme settings:

![GetImage](https://user-images.githubusercontent.com/122445919/216845808-3661c115-f726-4500-a7b5-5449abc6e954.png)

**Reply by texasmouse** [reply]
One moment, please…. Is ‘Themes’ a folder or a .yaml file?  I believe it is a folder, but want to confirm.

**Reply by novisys** [reply]
it's a folder that contain a .yaml file ...
